.env.example

```diff
@@ -6,6 +6,10 @@ REACT_APP_LOG_LEVEL=
 # Requires. Default: http://localhost:3003
 REACT_APP_SERVER_URL=
 
+# GoodDapp instance URL
+# Required. Default: http://localhost:3000
+REACT_APP_PUBLIC_URL=
```

Comment:
Add the default value to the `.env.dev` file.

`REACT_APP_PUBLIC_URL=http://localhost:3000`